app/routes/show.js

```diff
@@ -4,6 +4,7 @@ import RSVP from 'rsvp';
 
 export default class ShowRoute extends Route {
   @service resultProcessor;
+  @service store;
```

Comment:
Could you make a separate PR with this change? I know it's only a small line, but it'll help keep commits organized. Appreciated :)